<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogWindowInteractive</class>
 <widget class="QDialog" name="DialogWindowInteractive">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>144</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="pushButton_open">
     <property name="text">
      <string>打开子窗口</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="lineEdit_send"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_send">
       <property name="text">
        <string>发送</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="lineEdit_receive"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_receive">
       <property name="text">
        <string>接收</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
